### ISSUE TITLE
A11y_mssql for VSCode_mssql Extension_Connections_ScreenReader:Screen reader is announcing the position of "New Query" as "New Query 1 of 5", but not announcing positions of other menu items like  Disconnect, Remove, Refresh and Create Project from Database.

### ISSUE THREAD

**Issue by Yashi1201** [microsoft/vscode-mssql#17200]
## Actual:
Screen reader is announcing the position of "New Query" as "New Query 1 of 5", but not announcing positions of other menu items like  Disconnect, Remove, Refresh and Create Project from Database.

**Similar issue is also observed in the following:-**

1. mssql Extension-Context Menu actions:- ScreenReader is announcing the position of "Select Top 1000" as "Select Top 1000 1 of 5" but not announcing positions of other menu items like "Script as Create", "Script as Drop", "Script as Alter" and "Refresh".

## Expected:
Screen Reader should also announce the positions of Disconnect, Remove, Refresh and Create Project from Database menu item. It should announce them as "Disconnect 2 of 5", "Remove 3 of 5" and "Refresh 4 of 5" and "Create Project from Database 5 of 5."

**GitHubTags**:#A11yMAS;#A11yTCS;#BM_mssql for VSCode_Jan2022;#mssql for VSCode;#Win32;#FTP;#DesktopApp;#A11ySev2;#AINotInScope;#WCAG1.3.1;#NVDA;#Jaws;#STP;#Element:menuitem;

## Environment Details:
Application Name: mssql for VSCode
Windows Version: Win10
Screen Reader: NVDA(2021.3), Jaws(2022.2112.24)

## Repro Steps:

1. Open Visual studio code
2. Tab till extension button and search for "mssql"
3. Tab till install and move to the SQL Server button
4. Tab till Add Connection Button and insert the valid credentials
5. Once the database is added right click on the database name
6. Disconnect, Remove  and Refresh menu item will appear.
7. Run NVDA and tab to the "Disconnect" menuitem and check if the NVDA is announcing the position of Disconnect menu item.

## User Impact:
Screen reader users will get confused if the screen reader will not announce the positions of the menuitems.

“[Check out Accessibility Insights!](url) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Comment by smartguest** [comment]
This is also an issue with the dropdown menu for vscode in general, it's not just limited to right clicking on a database name, but rather it's also the toolbar on top of vscode as well and similar dropdowns menus. This needs to be investigated for vscode in general.

**Comment by smartguest** [comment]
Closing as this is an upstream issue with vscode and the design of how folder items are announced is under discussion at the moment: https://github.com/microsoft/vscode/issues/141316

**Comment by Yashi1201** [comment]
@smartguest This bug can be closed by trusted testers by C + AI team after bug verification. Can you please change the bug to open state. 
Can you please let us know if it is external to another product. If yes, to which component it is external to.

**Comment by Yashi1201** [comment]
@smartguest Can you please look into the above comment and change the bug state to open state. Thanks
